README.md

```diff
@@ -84,10 +118,10 @@ Initializing the SDK should be done in native code inside your `AppDelegate` (iO
       AEPMobileEdge.class,
       AEPMobileEdgeIdentity.class,
       AEPMobileEdgeConsent.class,
-      //AEPMobileIdentity.class,
-      //AEPMobileUserProfile.class,
-      //AEPMobileMessaging.class,
-      //AEPMobileAssurance.class,
+      //AEPMobileMessaging.class, //enable this extension for using messaging functionalities
```

Comment:
Ditto: enable this extension if you installed `@adobe/react-native-aepmessaging`